de.akaflieg_freiburg.enroute.json

```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "de.akaflieg_freiburg.enroute",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "enroute",
+    "finish-args": [
+	"--filesystem=host",
+        "--share=ipc",
+	"--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+	"--system-talk-name=org.freedesktop.GeoClue2",
```

Comment:
nitpick, but the indenting is inconsistent here and it's a bit annoying to look at ;-)